.codegen.json

```diff
@@ -14,6 +16,8 @@
             "go"
         ],
         "post_generate": [
+            "go test ./acceptance -v -update -run TestAccept/cmd/workspace || true",
```

Comment:
We run it 2 times, the first one to update the test configuration and output and second to make sure it succeeds